alkira/resource_alkira_connector_gcp_interconnect_helper.go

```diff
@@ -125,7 +89,7 @@ func expandGcpInterconnectInstances(in []interface{}, so []interface{}, m interf
 		if v, ok := cfgInstance["edge_availability_domain"].(string); ok {
 			newInstance.GcpEdgeAvailabilityDomain = v
 		}
-		if v, ok := cfgInstance["bgp_auth_key_alkira"].(string); ok {
+		if v, ok := cfgInstance["bgp_auth_key"].(string); ok {
```

Comment:
Is this intentional? I think we have both bgp_auth_key and bgp_auth_key_alkira...

Reply:
Yes , we only have `bgp_auth_key` field in this schema.